apps/src/studio/studio.js

```diff
@@ -4846,6 +4852,10 @@ Studio.askForInput = function (question, callback) {
   );
 };
 
+Studio.hideInputPrompt = function () {
+  // To be overriden when the prompt is displayed.
```

Comment:
Given that we (presumably) only ever want one prompt shown at once, what about if `askForInput` does something like `Studio.inputPrompt = target` (after perhaps checking to make sure that it isn't already assigned), and this method references and clears that variable?